src/controllers/SubredditsController.ts

```diff
@@ -1,14 +1,21 @@
-import { BodyParams, Controller, Post } from '@tsed/common';
-import { SubredditsCreation } from '../models/auth.types';
-
-import { SubredditsService } from '../services/SubredditsService';
-
+import { BodyParams, Controller, Get, Post, UseBefore } from "@tsed/common";
+import { Subreddits } from "../entities/Subreddits";
+import { AuthMiddleware } from "../middlewares/AuthMiddleware";
+import { SubredditsCreation } from "../models/auth.types";
+import { SubredditsService } from "../services/SubredditsService";
 
 @Controller('/')
-export class SubredditsController {
+export class UserController {
```

Comment:
I think that this should be named SubredditsController right?

Reply:
Yes, thank you! :D